src/I18n/I18nProvider.tsx

```diff
@@ -22,3 +22,5 @@ export const I18nProvider: FunctionComponent<I18nProviderProps> = ({
     {children}
   </I18nextProvider>
 );
+
+export * from "react-i18next";
```

Comment:
Thank you @kelvah . I have a comment to this change. Is it save to export `all` from an existing library? 

Shouldn't we at least guide users in documentation to not install `react-i18next` dependency?

or shoudln't  we maybe export just `useTranslation` ?

or maybe to not introduce another method with same name maybe we shouldn export just some wrapper like?

```
export appServicesUseTranslation = useTranslation();
```

Reply:
> Thank you @kelvah . I have a comment to this change. Is it save to export `all` from an existing library?
> 
> Shouldn't we at least guide users in documentation to not install `react-i18next` dependency?

I think I added this to the `README`. If it's not clear enough we can of course make it more explicit.

> or shoudln't we maybe export just `useTranslation` ?
> 
> or maybe to not introduce another method with same name maybe we shouldn export just some wrapper like?
> 
> ```
> export appServicesUseTranslation = useTranslation();
> ```

I was re-exporting only `useTranslation` initially. Then I've realized we also need the [`Trans` component](https://react.i18next.com/latest/trans-component) for smart events. So I thought it was easier to just re-export the whole library.

If someone needs something that's not exported or if a new relevant feature will be added in future releases of `react-i18next`, we'll have to make new `ui-components` releases every time. Maintaining a sub list of exports looks a bit annoying.